recipe/session/cookieAndHeaders.go

```diff
@@ -187,7 +193,7 @@ func setHeader(res http.ResponseWriter, key, value string, allowDuplicateKey boo
 	}
 }
 
-func setCookie(config sessmodels.TypeNormalisedInput, res http.ResponseWriter, name string, value string, expires uint64, pathType string) {
+func SetCookie(config sessmodels.TypeNormalisedInput, res http.ResponseWriter, name string, value string, expires uint64, pathType string) {
```

Comment:
- `setToken` has different casing
- I've added an extra debug log to the start of `setToken`: ```
    logDebugMessage(`setToken: Setting ${tokenType} token as ${transferMethod}`);```

Reply:
setToken still has different casing